src/ensembl/src/shared/components/app-bar/AppBar.scss

```diff
@@ -3,6 +3,7 @@
 .appBar {
   display: grid;
   grid-template-columns: minmax(0, 1fr) minmax(250px, auto);
+  align-content: start;
```

Comment:
Curiously, this little rule makes a difference on Safari between this (bad, currently in production — notice how species elements are pressed to the grey bar below them):
 
![image](https://user-images.githubusercontent.com/6834224/72920456-3a63a980-3d41-11ea-8417-7bf03151c22e.png)

and this (better):

![image](https://user-images.githubusercontent.com/6834224/72920516-5d8e5900-3d41-11ea-8f52-9043b18fd8d8.png)